extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -155,10 +159,20 @@ export class AmpAnalytics extends AMP.BaseElement {
   layoutCallback() {
     // Now that we are rendered, stop rendering the element to reduce
     // resource consumption.
+    this.element.getResources().setOwner(this.element,
+        this.element.parentElement);
     return this.ensureInitialized_();
   }
 
   /** @override */
+  unlayoutCallback() {
+    const ampDoc = this.getAmpDoc();
+    Transport.markCrossDomainIframeAsDone(ampDoc.win.document, this.type_);
```

Comment:
It will delete the frame when the (static) count of how many amp-analytics instances are using it reaches 0.